HLTrigger/Configuration/python/HLT_75e33/modules/hltDoubleMuon7DZ1p0_cfi.py

```diff
@@ -1,13 +1,14 @@
 import FWCore.ParameterSet.Config as cms
 
-hltDoubleMuon7DZ1p0 = cms.EDFilter("HLT2L1TkMuonL1TkMuonDZ",
+hltDoubleMuon7DZ1p0 = cms.EDFilter("HLT2L1P2GTCandL1P2GTCandDZ",
     MaxDZ = cms.double(1.0),
     MinDR = cms.double(-1),
     MinN = cms.int32(1),
-    inputTag1 = cms.InputTag("hltL1TkDoubleMuFiltered7"),
-    inputTag2 = cms.InputTag("hltL1TkDoubleMuFiltered7"),
-    originTag1 = cms.VInputTag("l1tTkMuonsGmt"),
-    originTag2 = cms.VInputTag("l1tTkMuonsGmt"),
+    algoBlockTag = cms.InputTag("l1tGTAlgoBlockProducer"),
```

Comment:
does this need to be done for every single path? The P2GT algo block exists once for each event.

Reply:
@SohamBhattacharya can you please reply here?

Reply:
> does this need to be done for every single path? The P2GT algo block exists once for each event.

@artlbv Do you mean whether the algoBlock InputTag needs to be specified for each path, instead of having `cms.InputTag("l1tGTAlgoBlockProducer")` as the default?

Reply:
yes. Essentially this is a per-run setting.

Reply:
I feel this comment is mixing things up. The InputTag is a parameter to be used to configure the collection to look at by the specific configured module instance. This has to be in every module instance if the module class looks at the collection. The Producer of the collection itself only needs to be run once in the overall configuration, or the product be on the input file, but even if you have the producer in each path, the FW ensures it is only run once.

Reply:
There is a default defined in the `fillDescriptions` method of this module, so I guess it's not *strictly* needed from the framework point of view to put the inputTag in all the path configuration. On the other hand we discussed some time ago internally in TSG and we prefer to have fully dumped configuration for the phase-2 setup. So I don't see issue with having them as they are (unless of course if @SohamBhattacharya wants to change the name of the parameter).

Reply:
> yes. Essentially this is a per-run setting.

Yes indeed. I'll change it.

Reply:
@Martin-Grunewald if e.g. we would have two P2GT collections (as now in Phase1: the unpacked uGT and the emulated one), we would need to change that tag in every single HLT path then. I don't think that this is convenient and fail prove. To me this GT collection name should be configured at the level of the L1 menu.

Reply:
Yes, you/we/STORM must change the InputTag if one wants to switch between unpacked vs emulated (all or some paths), that's the whole point of tracking which collection precisely was used in the event for what!
So the InputTag must NOT be hardwired!

Reply:
I am not sure what you mean by:

> I don't think that this is convenient and fail prove

that's the model used so far....

Reply:
(Yes it is a pain with the dump of the Ph-2 menu, but in the long run the Ph-2 HLT menu will be in ConfDb or its successor which will provide the tool for a global change of this parameter)

Reply:
This does not prevent to configure the GT collection name at the level of the L1T menu, but then the HLT menu needs to be configured to look at the desired L1T collection by configuring the InputTag.